client/asset/btc/btc.go

```diff
@@ -875,36 +881,46 @@ func newRPCWallet(requester RawRequesterWithContext, cfg *BTCCloneCFG, walletCon
 	return &ExchangeWalletFullNode{btc}, nil
 }
 
-func newUnconnectedWallet(cfg *BTCCloneCFG, walletCfg *WalletConfig) (*baseWallet, error) {
+func validateBtcWalletConf(walletCfg *WalletConfig) error {
+	var err error
 	// If set in the user config, the fallback fee will be in conventional units
 	// per kB, e.g. BTC/kB. Translate that to sats/byte.
-	fallbackFeesPerByte := toSatoshi(walletCfg.FallbackFeeRate / 1000)
-	if fallbackFeesPerByte == 0 {
-		fallbackFeesPerByte = cfg.DefaultFallbackFee
```

Comment:
Isn't the only thing we need to do is to check that 

```Go
if walletCfg.FallbackFeeRate > 0 && walletCfg.FallbackFeeRate < 1000 {
    return nil, fmt.Errorf("minimum fee rate is 1000 sats/kB")
}
```

Reply:
If we just check for `FallbackFeeRate` it will still be possible to create a wallet with "invalid" values, for example: 

someone tries to create a wallet with Highest acceptable fee rate less than the minimum value allowed (anything less than 1 sat/B), would create the wallet directories and db, but the form would throw an error making a wallet which can not be opened, like described in the issue.

Also, with `toSatoshi()` It rounds for the minimum value possible, so we just need to check if it is not 0, considering we are not checking for a maximum value.

Reply:
Notice that in @buck54321's example code it permits 0, allowing the default to be applied elsewhere (caller).

The validator function can still check each of the settings that can result in an unopenable wallet.

Reply:
So it does not set values on error, it sets if values are not specified. 

Thanks for clarifying chapp, I had misunderstood this. I fixed it now.

Reply:
> Isn't the only thing we need to do is to check that
> 
> ```go
> if walletCfg.FallbackFeeRate > 0 && walletCfg.FallbackFeeRate < 1000 {
>     return nil, fmt.Errorf("minimum fee rate is 1000 sats/kB")
> }
> ```

I'm saying this entire PR should be just these three lines. You didn't need to touch `WalletConfig` or create any new functions. We already had a pattern established for validating input values, We just weren't checking this condition.

Reply:
okay, but the problem is actually on the `Create` method, which does not validate the spv wallet allowing it to create a wallet which can not be opened. Should I copy this validations over that method?

Reply:
I see what you mean. It wasn't failing on creation but failed on open, which tanked the creation process. I guess wallets need to basically guarantee that the settings given to `Create` are valid. 

We should also re-consider the sequence in `CreateWallet` that allows this to happen. Should we somehow delete the wallet if the full `CreateWallet` sequence doesn't complete? Or should we store the wallet (in `db` and `wallets` map) right after `Create`, so users could have recovered by adjusting configuration settings.

Reply:
> We should also re-consider the sequence in `CreateWallet` that allows this to happen. Should we somehow delete the wallet if the full `CreateWallet` sequence doesn't complete?

If in `(*Core).CreateWallet` it creates the wallet fine but then opening it fails, I agree it would be ideal to "uncreate" the wallet.  I think that can be handled **if** the wallet is a `Recoverer` that allows it to be destroyed.  We can follow-up with that I think.  Ideally the `Move` method that started out as `Destroy` if I recall correctly, would take an empty string to signal deletion instead of backup.